bitbots_ros_control/scripts/foot_pressure_tester.py

```diff
@@ -1,143 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Are these test scripts so irrelevant, that we want to remove them entirely? Don't we want to migrate them over to ROS 2? @jgueldenstein @timonegk 

Also concerns imu_interactive_markers.